src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -2869,6 +2869,31 @@ public static unsafe void ConstructorsTest_InvalidUTF8()
             }
         }
 
+        [Fact]
```

Comment:
This will need `[SkipOnTargetFramework(~TargetFrameworkMonikers.NetFramework)]` to get CI green because of the behavior on .NET Framework is different.